apps/gp2-server/src/routes/user.route.ts

```diff
@@ -68,6 +70,14 @@ export const userRouteFactory = (userController: UserController): Router =>
 
         // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
         const loggedInUserId = req.loggedInUser!.id;
+
+        if (
+          req.loggedInUser?.onboarded !== true &&
```

Comment:
isn't this exactly the same what is already in permissionHandler, which is being injected here anyway?